examples/hosts/iframe-host/src/inner.ts

```diff
@@ -24,26 +28,36 @@ async function getFluidObjectAndRender(container: Container, div: HTMLDivElement
     view.render(div, { display: "block" });
 }
 
-export async function runInner(divId: string) {
-    const div = document.getElementById(divId) as HTMLDivElement;
+async function loadFluidObject(
+    divId: string,
+    documentId: string,
+    createNew: boolean,
+) {
+    const componentDiv = document.getElementById(divId) as HTMLDivElement;
 
     const documentServiceFactory = await InnerDocumentServiceFactory.create();
-    const module = { fluidExport: TodoContainer };
-    const codeLoader = { load: async () => module };
 
-    const loader = new Loader({
-        urlResolver: documentServiceFactory.urlResolver,
-        documentServiceFactory,
-        codeLoader,
-    });
+    const urlResolver = new InsecureTinyliciousUrlResolver();
```

Comment:
>InsecureTinyliciousUrlResolver [](start = 28, length = 30)

the inner should never use real drivers. it should proxy everything out to the outer driver